### ISSUE TITLE
Fix roadway validation test error from read/write

### ISSUE THREAD

**Issue by e-lo** [wsp-sag/network_wrangler#111]
It looks like the roadway network that we write doesn't pass validation upon being read back in?

```
tests/test_roadway.py::test_roadway_read_write
--Starting: test_roadway_read_write
Writing to: /Users/josiekressner/tf/network_wrangler/scratch
FAILED

E               SystemExit: RoadwayNetwork: Data doesn't conform to schema
```

**Comment by kulshresthaa** [comment]
it's not just roadway network that we write, existing stpaul roadway network also doesn't conform to validation schema (when we change fast to False). Everywhere fast is True, so the reading existing network skips validation for all of the tests!

**Comment by kulshresthaa** [comment]
![image](https://user-images.githubusercontent.com/13478750/67425342-a8ef6a80-f58c-11e9-8cf3-3bc905879923.png)

**Comment by kulshresthaa** [comment]
@e-lo How does the link schema handles None values in the link data? `access`, `area`, `bridge`, `ref`, `service`, `tunnel` etc does not have values for the links...

**Comment by josiekre** [comment]
This is blocking my tests on the transit now too since I need to read in the RoadwayNetwork for the nx graph.

**Comment by e-lo** [comment]
You can read in with the parameter fast=True and it will skip validation

**Comment by kulshresthaa** [comment]
I have updated the link schema to include null to be specified for some values. I changed it for all the fields which are null in the current network. 

Had to update type for all these fields - `access`, `area`, `bridge`, `junction`, `tunnel`, `service`, `width`, `ref`, `osmid`, `oneway`, `name`, `maxspeed`, `OSMLanes`, `key`, and  `highway`!

We can decide later if any of above fields should be mandatory and not null. 

Tests are passing with fast=False
